openmeter/productcatalog/subscription/http/get.go

```diff
@@ -62,3 +64,65 @@ func (h *handler) GetSubscription() GetSubscriptionHandler {
 		)...,
 	)
 }
+
+type (
+	ListCustomerSubscriptionsParams = struct {
+		CustomerID string
+		Params     api.ListCustomerSubscriptionsParams
+	}
+	ListCustomerSubscriptionsRequest = struct {
+		CustomerID models.NamespacedID
+		Page       pagination.Page
+	}
+	ListCustomerSubscriptionsResponse = pagination.PagedResponse[api.Subscription]
+	ListCustomerSubscriptionsHandler  = httptransport.HandlerWithArgs[ListCustomerSubscriptionsRequest, ListCustomerSubscriptionsResponse, ListCustomerSubscriptionsParams]
+)
+
+func (h *handler) ListCustomerSubscriptions() ListCustomerSubscriptionsHandler {
+	return httptransport.NewHandlerWithArgs(
+		func(ctx context.Context, r *http.Request, params ListCustomerSubscriptionsParams) (ListCustomerSubscriptionsRequest, error) {
+			ns, err := h.resolveNamespace(ctx)
+			if err != nil {
+				return ListCustomerSubscriptionsRequest{}, fmt.Errorf("failed to resolve namespace: %w", err)
+			}
+
+			return ListCustomerSubscriptionsRequest{
+				CustomerID: models.NamespacedID{
+					Namespace: ns,
+					ID:        params.CustomerID,
+				},
+				Page: pagination.NewPageFromRef(params.Params.Page, params.Params.PageSize),
+			}, nil
+		},
+		func(ctx context.Context, req ListCustomerSubscriptionsRequest) (ListCustomerSubscriptionsResponse, error) {
+			var def ListCustomerSubscriptionsResponse
+
+			subs, err := h.SubscriptionService.List(ctx, subscription.ListSubscriptionsInput{
```

Comment:
It's fine to use List here without any ordering args, but let's add at least a [default ordering](
https://github.com/openmeterio/openmeter/blob/232109fd0804883ae92dfee9b568d25a28773bab/openmeter/subscription/repo/subscriptionrepo.go#L170) here or the ordering might not be deterministic.